packages/apps/fortune/exchange-oracle/server/src/common/guards/signature.auth.ts

```diff
@@ -27,7 +27,13 @@ export class SignatureAuthGuard implements CanActivate {
     );
     if (!roles) throw new NotImplementedException(ErrorSignature.MissingRoles);
     const request = context.switchToHttp().getRequest();
-    const data = request.body;
+    const { escrow_address, ...restBody } = request.body;
```

Comment:
We should the lower case conversion in the signature verification (method verifySignature). If we do it here and we call verifySignature from somewhere else the issue will persist. So this changes here are not required